HISTORY.markdown

```diff
@@ -1,5 +1,30 @@
 # History
 
+### 2.0.0, 
+
+*Breaking compatibility changes in this release.*
+
+#### New features and fixes
+
+* Image resize on client before uploading (`image-shrink` option).
+* Autostore option was removed (but not autostore itself).
+  Now file is automatically stored if project allows autostore.
+* New option `do-not-store` is added when autostore for your project is enabled
```

Comment:
new option do-not-store. use it when autostore is enabled in project but you don't want to store files uploaded with particular widget(s)